### PR TITLE
Update Run2022 relval wfs

### PR DESCRIPTION
**Pull request by kskovpen**
#### PR description:

Update Run2022 data wfs for release validation in preparation for the upcoming data reprocessing campaign.

#### PR validation:

This update is similar to #39265. The validation will be triggered upon the progress on the corresponding [transfer request](https://its.cern.ch/jira/projects/CMSTRANSF/issues/CMSTRANSF-437) for the input RAW data to CERN T2.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. 

We will need this PR backported to 12_4_X and merged in the successor of 12_4_10.

### CODE REVIEW

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -448,26 +448,52 @@
...
-workflows[140.020] = ['',['RunBTagMu2022C','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-
-workflows[140.031] = ['',['RunZeroBias2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.032] = ['',['RunJetMET2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.033] = ['',['RunEGamma2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.034] = ['',['RunMuon2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.035] = ['',['RunTau2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
-workflows[140.036] = ['',['RunBTagMu2022D','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
+
+workflows[140.001] = ['',['RunMinimumBias2022A','HLTRUN3','RECONANORUN3','HARVESTRUN3']]
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_standard.py:452]: hi, in light of the discussion going at https://its.cern.ch/jira/browse/PDMVRELVALS-155 I wonder if it's possible to profit of this PR to change the `RECONANORUN3` to include also standard DQM sequences:

https://github.com/cms-sw/cmssw/blob/6ba5904a9334bdc20ece0528a157efad0c20b699/Configuration/PyReleaseValidation/python/relval_steps.py#L2737

Comment (kskovpen) [reply Configuration/PyReleaseValidation/python/relval_standard.py:452]: Thanks, done.

**Comment by kskovpen** [comment]
@cmsbuild please test

**Comment by kskovpen** [comment]
@cmsbuild please test

**Comment by kskovpen** [comment]
@cmsbuild please test

**Comment by mmusich** [comment]
is the target of this `CMSSW_12_6_0_pre5`? What is still outstanding?
It would be good to converge for the benefit of https://its.cern.ch/jira/browse/PDMVRELVALS-155 @kskovpen @sunilUIET

**Comment by kskovpen** [comment]
The transfer was completed today: https://its.cern.ch/jira/projects/CMSTRANSF/issues/CMSTRANSF-437?filter=allopenissues

**Comment by kskovpen** [comment]
@cmsbuild please test

**Comment by kskovpen** [comment]
@cmsbuild please test

**Comment by kskovpen** [comment]
+pdmv

**Comment by srimanob** [comment]
@missirol 
Should we merge this one first, then you update your PR?

**Comment by srimanob** [comment]
+Upgrade

**Comment by perrotta** [comment]
+1

**Pull request merged**